Yoast/wordpress-seo
# Add tests for PR #10162

## Summary

This PR can be summarized in the following changelog entry:

* Adds unit-tests for #10162.

## Relevant technical choices:

* Decided not to write tests for `class-wpseo-custom-fields.php` and `class-wpseo-custom-taxonomies.php` as requested in the issue because it is mainly WordPress code.

## Test instructions

This PR can be tested by following these steps:

* Make sure the PHP unit tests run.

## Quality assurance


Fixes #10180 
Fixes #10321
